x-pack/plugins/security_solution/public/resolver/store/data/reducer.ts

```diff
@@ -101,6 +105,33 @@ export const dataReducer: Reducer<DataState, ResolverAction> = (state = initialS
       relatedEvents: new Map([...state.relatedEvents, [action.payload.entityID, action.payload]]),
     };
     return nextState;
+  } else if (action.type === 'serverRequestedCurrentRelatedEventData') {
```

Comment:
Would you prefix this with `app` instead of `server`?

Reply:
copypasta thanks!